packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -69,6 +69,12 @@ type AlertProps = {
    */
   onDismiss?: () => void;
 
+  /**
+   * Can be used to render an icon
+   */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  leading?: ComponentType<any>;
```

Comment:
```suggestion
  icon?: ComponentType<IconProps>;
```

I don't remember if this is the correct type or not 🤔 Just check once in Button component

Reply:
We can use `IconComponent` type